src/Conda.jl

```diff
@@ -70,7 +70,16 @@ function _set_conda_env(cmd)
         pop!(env, var)
     end
     env["CONDARC"] = CONDARC
-    setenv(cmd, env)
+    env
+end
+
+"""
+Use a cleaned up environment for the command `cmd`.
+
+Any environment variable starting by CONDA will interact with the run.
+"""
+function _set_conda_env(cmd)
```

Comment:
if so, it's need to refactor with the newer function like `run_with_env`.
```julia
run_with_env(`$conda install -y $pkg`)

run_with_env(`$conda remove -y $pkg`)
```

instead of
```julia
withenv(_get_conda_env()...) do
    run(`$conda install -y $pkg`)
end

run(_set_conda_env(`$conda remove -y $pkg`))
```